frontend/src/constants/queryKeyConstants.ts

```diff
@@ -0,0 +1,14 @@
+export const queryKeyConstants = {
+  courseAndMaterials: 'CourseAndMaterials',
+  courseInfo: 'CourseInfo',
+  paginatedCoursesList: 'paginatedCoursesList',
+  myCourses: 'myCourses',
+  employeeProfile: 'EmployeeProfile',
+  courseRequests: 'courses-requests',
+  employeesLis: 'employees-list',
+  profile: 'profile',
+  clientCourseInfo: 'ClientCourseInfo',
+  paginatedClientCoursesList: 'paginatedClientCoursesList',
+  courseTest: 'CourseTest',
+  getTestTime: 'getTestTime',
+};
```

Comment:
```suggestion
export const QUERY_KEYS = {
    courseInfo: 'CourseInfo',
    paginatedCoursesList: 'PaginatedCoursesList',
  ....
};
```

it is better to write data in the same style.

Reply:
agree with @ostapdans99